### PR TITLE
do not validate attachment values

### PR DESCRIPTION
**Pull request by FynnBe**

### CODE REVIEW

File: bioimageio/spec/v0_3/schema.py [hunk 1]
```diff
@@ -84,7 +84,7 @@ class RDF(BioImageIOSchema):
 
     attachments = fields.Dict(
         fields.String,
-        fields.Union([fields.URI(), fields.List(fields.URI)]),
+        # fields.Union([fields.URI(), fields.List(fields.URI)]),  # do not validate attachment values; todo: clean up
```
Comment (oeway) [inline bioimageio/spec/v0_3/schema.py:87]:
```suggestion
        fields.List(fields.Union(fields.URI, fields.Dict())),
```

Comment (oeway) [reply bioimageio/spec/v0_3/schema.py:87]: A minor suggestion, or we can do without any validation.

Comment (FynnBe) [reply bioimageio/spec/v0_3/schema.py]: good idea! 👍

Comment (oeway) [reply bioimageio/spec/v0_3/schema.py]: Oops, we need to fill in something in `fields.list(..)`, the CI fails, sorry.

**Review by oeway — APPROVED** [review]
Looks good, thanks for fixing this!

**Pull request merged**